src/gt4py/config.py

```diff
@@ -36,13 +36,19 @@
 
 GT_INCLUDE_PATH: str = os.path.abspath(os.path.join(GT_REPO_PATH, "include"))
 
+# OpenMP preprocessor and link flags
+OPENMP_CPPFLAGS: str = os.environ.get("OPENMP_CPPFLAGS", "-fopenmp")
+OPENMP_LDFLAGS: str = os.environ.get("OPENMP_LDFLAGS", "")
+
 # Settings dict
 build_settings: Dict[str, Any] = {
     "boost_include_path": os.path.join(BOOST_ROOT, "include"),
     "cuda_bin_path": os.path.join(CUDA_ROOT, "bin"),
     "cuda_include_path": os.path.join(CUDA_ROOT, "include"),
     "cuda_library_path": os.path.join(CUDA_ROOT, "lib64"),
     "gt_include_path": os.environ.get("GT_INCLUDE_PATH", GT_INCLUDE_PATH),
+    "openmp_cppflags": OPENMP_CPPFLAGS,
```

Comment:
Since there is no need to preprocess relative paths or something like that, I would directly set the value of the settings in the dict to simplify the code: 

```
"openmp_cppflags": os.environ.get("OPENMP_CPPFLAGS", "-fopenmp")
"openmp_ldflags": os.environ.get("OPENMP_LDFLAGS", "")
```